charts/base-cluster/README.md

```diff
@@ -207,6 +207,9 @@ This helm chart requires flux v2 to be installed (https://fluxcd.io/docs/install
 | rbac.view.users | list | `[]` |  |
 | speedtest.enabled | bool | `true` |  |
 | speedtest.host | string | `"speedtest"` |  |
+| speedtest.image.registry | string | `""` |  |
+| speedtest.image.repository | string | `"adolfintel/speedtest"` |  |
+| speedtest.image.tag | string | `"latest"` |  |
```

Comment:
Something is wrong, this shouldn't be a change in here?

Reply:
Yesterday some parts mixed with main branch (and default commit message become incompatible on validation step) so I've tryed to restore it, but in same time extra changes (chore) applied on all branches.
Currently 'md' is in sync